tests/lib/snaps/test-snapd-devpts/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-devpts
+version: 1.0
+apps:
+    openpty:
+        command: bin/openpty
+        plugs: [ physical-memory-observe ]
```

Comment:
Why do we need pyhsical-memory-observe here?

Reply:
We use physical-memory-observe because it is an interface with a device that is going to be present on all systems. We don't access it though, we just want the interface so we can trigger the device cgroup.